scale/job/models.py

```diff
@@ -3019,8 +2160,6 @@ def get_job_types_v6(self, keywords=None, ids=None, is_active=None, is_system=No
             key_query = Q()
             for keyword in keywords:
                 key_query |= Q(name__icontains=keyword)
-                key_query |= Q(title__icontains=keyword)
```

Comment:
Need to update the docs as I think they discuss the keyword searching titles and descriptions

Reply:
Documentation updated. Good catch.